.github/workflows/api-breaking-changes.yaml

```
@@ -0,0 +1,190 @@
+# Generate an API changelog by diffing the OpenAPI schema between base and HEAD.
+name: API Breaking Changes Check
+
+on:
+  pull_request:
+    branches:
+      -- '**'
```

actionlint (1.7.12)

[error] character ' ' is invalid for branch and tag names. ref name cannot contain spaces, ~, ^, :, [, ?, *. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)